src/js/tabs/power.js

```diff
@@ -247,7 +253,11 @@ TABS.power.initialize = function (callback) {
                 for (var i = 0; i < VOLTAGE_METERS.length; i++) {
                     var elementName = '#voltage-meter-' + i + ' .value';
                     var element = $(elementName);
-                    element.text(i18n.getMessage('powerVoltageValue', [VOLTAGE_METERS[i].voltage]));
+                    if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
```

Comment:
@mikeller So I leave like this or revert to the original?

Reply:
These are meant to be voltage meters for different voltages (5V, 3.3V, individual cells), so they should keep their original reading in low resolution. I was thinking about replacing the main batter voltage in the Battery tab with `ANALOG.voltage`.

Reply:
There is the BATTERY_STATE voltage (in 0.01V) in the Power state tab